ts/displayTodos.ts

```diff
@@ -0,0 +1,13 @@
+import { addTodo } from "./addTodo.js";
+import { tasks } from "./todoapp.js";
+import { isTypeTodo } from "./setSingleTodoDeletionListener.js";
```

Comment:
Why `displayTodos` should know anything about `setSingleTodoDeletionListener`. How does logic of deletion and displaying todos are connected ? This is wrong from the architectural perspective